trento-docs-site/modules/developer/nav_developer.adoc

```diff
@@ -35,69 +35,10 @@
-**** xref:ROOT:agent:Development/how-to-make-a-release.adoc[How to release a new version of Trento Agent]
-
-*** xref:ROOT:ansible:README.adoc[Trento Ansible]
-**** xref:ROOT:ansible:local-development-environment.adoc[Local Development Environment]
-
-*** xref:ROOT:checks:README.adoc[Trento Checks]
-
-*** xref:ROOT:helm-charts:README.adoc[Trento Helm Charts]
-**** xref:ROOT:helm-charts:cert-manager.adoc[Adding cert-manager for HTTPS]
-**** xref:ROOT:helm-charts:rabbitmq-tls.adoc[Generate TLS certificates for RabbitMQ]
-**** xref:ROOT:helm-charts:trento-server.adoc[Trento Server Helm Chart]
-
-*** xref:ROOT:web:README.adoc[Trento Web]
-**** xref:ROOT:web:monitoring/monitoring.adoc[Monitoring]
-**** xref:ROOT:web:alerting/alerting.adoc[Alerting]
-**** Authentication
-***** xref:ROOT:web:authentication/jwt_specification.adoc[JWT]
-***** xref:ROOT:web:authentication/spa_flow.adoc[Trento Single Page Application]
-**** Integration
-***** xref:ROOT:web:integration/oidc.adoc[OpenID Connect]
-**** Development
-***** xref:ROOT:web:development/environment_variables.adoc[Environment Variables]
-***** xref:ROOT:web:development/hack_on_the_trento.adoc[Hack on the Trento Web]
-
-*** xref:ROOT:wanda:README.adoc[Trento Wanda]
-**** Checks development
-***** xref:ROOT:wanda:specification.adoc[Checks Specification]
-***** xref:ROOT:wanda:expression_language.adoc[Expression Language]
-***** xref:ROOT:wanda:gatherers.adoc[Gatherers]
-**** Hack on Wanda
-***** xref:ROOT:wanda:development/demo.adoc[Wanda Demo]
-***** xref:ROOT:wanda:development/hack_on_wanda.adoc[Hack on Wanda]
-**** xref:ROOT:wanda:rhai_expressions_cheat_sheet.cheat.adoc[Rhai expressions cheatsheet]
-
-*** xref:ROOT:workbench:README.adoc[Workbench - Trento Agent Operators]
-
-*** xref:ROOT:contracts:README.adoc[Trento Contracts]
-
-*** xref:ROOT:tlint:README.adoc[TLint]
-**** xref:ROOT:tlint:www/README.adoc[TLint web]
-
-*** xref:ROOT:photofinish:README.adoc[Trento Photofinish]
-
-*** xref:ROOT:support:README.adoc[Trento Support Utilities]
-
-*** xref:ROOT:continuous-delivery:README.adoc[Continuous delivery project]
-
-*** xref:ROOT:werkzeugkoffer:README.adoc[werkzeugkoffer - Trento infrastructure & development tools]
-**** xref:ROOT:werkzeugkoffer:demo_idp/README.adoc[demo-idp Playbook - Provision a Keycloak IDP]
-
-*** xref:ROOT:mcp-server:README.adoc[Trento MCP Server]
-**** xref:ROOT:mcp-server:docs/README.adoc[Trento MCP Server documentation]
-***** xref:ROOT:mcp-server:docs/integration-suse-ai.adoc[SUSE AI integration]
-***** xref:ROOT:mcp-server:docs/integration-vscode.adoc[VS Code integration]
-***** xref:ROOT:mcp-server:docs/configuration-options.adoc[Configuration options]
-**** xref:ROOT:mcp-server:docs/developer/README.adoc[Trento MCP Server developer documentation]
-***** xref:ROOT:mcp-server:docs/developer/getting-started.adoc[Getting started with Trento MCP Server]
\ No newline at end of file
+include::partial$nav_components.adoc[]
```

Comment:
This is the cool part, from now on no need to adjust manually the content of upstream projects :)